core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsAddress.java

```diff
@@ -77,7 +77,7 @@ public void doWriteObject(ObjectOutput output, JGroupsAddress address) throws IO
       public JGroupsAddress doReadObject(ObjectInput unmarshaller) throws IOException, ClassNotFoundException {
          try {
             org.jgroups.Address address = org.jgroups.util.Util.readAddress(unmarshaller);
-            return (JGroupsAddress) JGroupsAddressCache.fromJGroupsAddress(address);
+            return (JGroupsAddress) JGroupsAddressCache.fromJGroupsAddressUnchecked(address);
```

Comment:
well, it doesn't make sense. it doesn't explain
1. why isn't failing in master
2. why the Marshaller (and externalizers) is being loaded for a non-clustered global configuration